### ISSUE TITLE
Tests fail with Mojolicious 7

### ISSUE THREAD

**Issue by eserte** [sshaw/Mojolicious-Plugin-MoreUtilHelpers#3]
See subject. A sample fail report:
http://www.cpantesters.org/cpan/report/3813800e-4e40-11e6-9c90-eb5010d7a009

Statistical analysis suggests that the failure happens with Mojolicious 7.0 (negative theta means "bad"):

```
****************************************************************
Regression 'mod:Mojolicious'
****************************************************************
Name                   Theta          StdErr     T-stat
[0='const']           1.0000          0.0000    57517306866459256.00
[1='eq_6.08']         0.0000          0.0000       0.00
[2='eq_6.11']         0.0000          0.0000       0.85
[3='eq_6.17']         0.0000          0.0000       0.00
[4='eq_6.19']         0.0000          0.0000       0.61
[5='eq_6.27']         0.0000          0.0000       0.85
[6='eq_6.31']         0.0000          0.0000       0.61
[7='eq_6.32']         0.0000          0.0000       0.61
[8='eq_6.34']         0.0000          0.0000       1.23
[9='eq_6.36']         0.0000          0.0000       0.61
[10='eq_6.37']       -0.0000          0.0000      -0.61
[11='eq_6.38']        0.0000          0.0000       1.23
[12='eq_6.41']        0.0000          0.0000       0.61
[13='eq_6.56']        0.0000          0.0000       0.00
[14='eq_6.58']        0.0000          0.0000       1.71
[15='eq_6.60']        0.0000          0.0000       0.00
[16='eq_6.62']        0.0000          0.0000       2.05
[17='eq_7.0']        -1.0000          0.0000    -39406187629459480.00

R^2= 1.000, N= 73, K= 18
****************************************************************
```

**Comment by eserte** [comment]
Probably relevant change: https://metacpan.org/source/SRI/Mojolicious-7.0/Changes#L8

**Comment by sshaw** [comment]
Fixed by #4.
